docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -24,7 +24,9 @@ license: |
 The `GROUP BY` clause is used to group the rows based on a set of specified grouping expressions and compute aggregations on
 the group of rows based on one or more specified aggregate functions. Spark also supports advanced aggregations to do multiple
 aggregations for the same input record set via `GROUPING SETS`, `CUBE`, `ROLLUP` clauses.
-When a FILTER clause is attached to an aggregate function, only the matching rows are passed to that function.
+These grouping analytics clauses can be specified with regular grouping expressions (partial grouping analytics) and 
+the grouping analytics clauses can be specified together (concatenated groupings).
```

Comment:
The grouping expressions and advanced aggregations can be mixed in the `GROUP BY` clause. See more details in the `Mixed Grouping Analytics` section.

Reply:
> The grouping expressions and advanced aggregations can be mixed in the `GROUP BY` clause. See more details in the `Mixed Grouping Analytics` section.

Done